src/SentryTarget.php

```diff
@@ -87,13 +100,41 @@ protected function getContextMessage()
         return '';
     }
 
+    /**
+     * @inheritdoc
+     */
+    public function collect($messages, $final)
+    {
+        foreach ($messages as $message) {
+            list($text, $level, $category, $timestamp, $traces) = $message;
+
+            [$message, $tags, $exception, $type, $extra] = $this->parseText($text);
+
+            $metadata = [];
+            if(!empty($tags))
```

Comment:
Thanks for your work!
Please format code according to [PSR-12](https://www.php-fig.org/psr/psr-12/) (spaces and braces)